pyriemann/spatialfilters.py

```diff
@@ -268,18 +270,20 @@ class CSP(BilinearFilter):
     References
     ----------
     [1] Zoltan J. Koles, Michael S. Lazar, Steven Z. Zhou. Spatial Patterns
-        Underlying Population Differences in the Background EEG. Brain
-        Topography 2(4), 275-284, 1990.
+    Underlying Population Differences in the Background EEG. Brain Topography
+    2(4), 275-284, 1990.
```

Comment:
I would revert these changes and make it so that references can be cross-link within the docstrings. See an example in :

https://github.com/scikit-learn/scikit-learn/blob/main/sklearn/svm/_classes.py#L635

you need the `..`  and then you can reference with `[1]_`